single-solution/SolutionPage.js

```diff
@@ -98,8 +129,8 @@ function fill_fields(){
         $("#solute_span2").html(solute_compound.formula());
         $("#solute_span3").html(solute_compound.formula());
 
-        $("#volume_span1").html(total_volume * 1000);
-        $("#volume_span2").html(total_volume * 1000);
+        $("#volume_span1").html(total_volume);
+        $("#volume_span2").html(total_volume);
```

Comment:
Instead of gathering the input in liters, I will gather based on milli Liters. Keep in mind that calculator.js requires liters, so external references will need to change to reflect this.